tests/test_geom_operations.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import numpy as np
+
+import aimsprop as ai
+
+# 1, 2, 3 should be done for basically all tests.
+
+
+def test_geom_operations(tmp_path):
```

Comment:
You don't use `tmp_path` in your function so it is not needed :)

This is a way to ask `pytest` to pass you a temporary directory you can use on a machine for saving scratch files to.

Reply:
Replaced with the generic trajectory pass

Reply:
temp_path has been removed